boot/linux-x86_64/crt0.S

```diff
@@ -0,0 +1,87 @@
+//------------------------------------------------------------------------------
+// Copyright (C) 2018-2019 Daedalean AG
+//
+// This library is free software; you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation; either version 3, or (at your option)
+// any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// Under Section 7 of GPL version 3, you are granted additional
+// permissions described in the GCC Runtime Library Exception, version
+// 3.1, as published by the Free Software Foundation.
+//
+// You should have received a copy of the GNU General Public License and
+// a copy of the GCC Runtime Library Exception along with this program;
+// see the files LICENSE and LICENSE.RUNTIME respectively.  If not, see
+// <http://www.gnu.org/licenses/>.
+//------------------------------------------------------------------------------
+
+//------------------------------------------------------------------------------
+// This will be run in a hosted, but freestanding, Linux environment
+//------------------------------------------------------------------------------
+
+#include "libsupcxx/boot/linux-x86_64/syscall.h"
+.section .text
+
+.global _start
+.type _start, @function
+
+.global _syscall
+.type _syscall, @function
+
+_start:
+  // TODO is this too far from call?
```

Comment:
Too far from call?

Reply:
Ah good catch, should have got rid of the comment